products/sle/main.pm

```diff
@@ -969,7 +969,12 @@ sub load_fips_tests_crypt() {
     loadtest "console/cryptsetup";
     loadtest "console/ecryptfs_fips";
 }
-
+sub load_patching_tests() {
+    loadtest 'boot/boot_to_desktop_sym';
+    loadtest "update/patch_before_migration";
+    loadtest "console/consoletest_finish_sym";
+    loadtest "x11/reboot_and_install";
+}
```

Comment:
please use only 'single ticks' with loadtest unless you want to use variable expansion